lib/influxer/metrics/metrics.rb

```diff
@@ -162,10 +170,50 @@ def tags
       @attributes.select { |k, _| tag_names.include?(k.to_s) }
     end
 
-    attributes :timestamp
+    def time(parse = false)
```

Comment:
I think, we can transform timestamp value only when building write data. Then we can use simple `attr_accessor` for `timestamp`.